src/APIM_ARMTemplate/apimtemplate/apimtemplate.csproj

```diff
@@ -8,11 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
I'm not sure why these folders were removed in the first place, but Visual Studio automatically removed the models folder when I re-introduced it and moved `ExtractorConfiguration.cs` there.